core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -873,493 +987,608 @@ public void testFormat()
                         Optional.empty(),
                         Optional.empty(),
                         Optional.empty(),
                         Optional.empty(),
-                        Optional.of(new Offset(new LongLiteral("2"))),
+                        Optional.of(new Offset(location(1, 43), new LongLiteral(location(1, 50), "2"))),
                         Optional.empty()));
     }
 
     @Test
     public void testSelectWithFetch()
     {
-        assertStatement("SELECT * FROM table1 FETCH FIRST 2 ROWS ONLY",
-                simpleQuery(
-                        selectList(new AllColumns()),
-                        new Table(QualifiedName.of("table1")),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.of(new FetchFirst(new LongLiteral("2")))));
-
-        assertStatement("SELECT * FROM table1 FETCH NEXT ROW ONLY",
-                simpleQuery(
-                        selectList(new AllColumns()),
-                        new Table(QualifiedName.of("table1")),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.of(new FetchFirst(Optional.empty()))));
-
-        Query valuesQuery = query(values(
-                row(new LongLiteral("1"), new StringLiteral("1")),
-                row(new LongLiteral("2"), new StringLiteral("2"))));
-
-        assertStatement("SELECT * FROM (VALUES (1, '1'), (2, '2')) FETCH FIRST ROW ONLY",
-                simpleQuery(
-                        selectList(new AllColumns()),
-                        subquery(valuesQuery),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.of(new FetchFirst(Optional.empty()))));
-
-        assertStatement("SELECT * FROM (VALUES (1, '1'), (2, '2')) FETCH FIRST ROW WITH TIES",
-                simpleQuery(
-                        selectList(new AllColumns()),
-                        subquery(valuesQuery),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
-                        Optional.empty(),
+        assertThat(statement("SELECT * FROM table1 FETCH FIRST 2 ROWS ONLY")).isEqualTo(simpleQuery(location(1, 1),
+                selectList(location(1, 1),
+                        new AllColumns(location(1, 8), Optional.empty(), ImmutableList.of())),
+                new Table(location(1, 15), qualifiedName(location(1, 15), "table1")),
+                Optional.empty(),
+                Optional.empty(),
```

Comment:
Please reformat so that either all arguments are in the same line, or in separate lines.
Apply it to other methods too.